mne/beamformer/_compute_beamformer.py

```diff
@@ -271,11 +282,21 @@ def _compute_beamformer(G, Cm, reg, n_orient, weight_norm, pick_ori,
                     assert Ck.shape[1:] == (3, 3)
                     # Invert for all dipoles simultaneously using matrix
                     # inversion.
```

Comment:
@larsoner would it make sense to refactor this with the code in ``_normalize_weights`` to make things easier down the line? Just thinking of our latest issue with two different paths for the orientation selection ...

Reply:
Yep it cleaned things up, pushing now

Reply:
Awesome, I will have a look later!